src/Image.cpp

```diff
@@ -54,7 +54,7 @@ void Image::Initialize()
             else
             {
                 for (auto c : pdb_info->Guid)
-                    spdlog::info("{:X}", (uint32_t)c);
+                    Logger::ErrorToMainFmt("{:X}", (uint32_t)c);
```

Comment:
I won't merge this stuff, this is no how logging should be done, the function names are way too specialized